packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -99,6 +101,42 @@ class AnalyzeCommand extends FlutterCommand {
     argParser.addFlag('fatal-warnings',
         help: 'Treat warning level issues as fatal.',
         defaultsTo: true);
+
+    // Android analysis sections. DO NOT add other arg after this line.
+    addAndroidAnalysis(argParser, verboseHelp);
+  }
+
+  static void addAndroidAnalysis(ArgParser argParser, bool verboseHelp) {
```

Comment:
are you going to call this from other sub-commands? If so maybe this should live in `flutter_command.dart` as an instance method?

Reply:
Actually, not sure this would make sense in other sub-commands. Are you just throwing this in a function to make it clear these are separate?

Reply:
yes that was the intention.